tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -55,7 +55,7 @@ def test_simple(self):
         )
         # TODO(dcramer): we need to pare this down -- lots of duplicate queries
         # for membership data
-        with self.assertNumQueries(34, using='default'):
+        with self.assertNumQueries(35, using='default'):
```

Comment:
Is this because we added another test case? This seems a little bizarre... is it meant to be some sort of proxy for the amount of queries a page makes?

Reply:
Yes it appears so. We did add one more database call. https://django.readthedocs.io/en/1.6.x/topics/testing/tools.html?highlight=assertnumqueries#django.test.TransactionTestCase.assertNumQueries